synapse/config/_base.py

```diff
@@ -201,6 +201,26 @@ def load_config(cls, description, argv):
 
```

Comment:
why don't we move `load_config` to a global function called `load_config_for_worker_app`?

Reply:
Could do, though I tend to think of the `load_config*` functions as creators of `HomeserverConfig`, so to me feels natural they'd be part of the class.